README.md

```diff
@@ -151,6 +151,31 @@ for more info.
 To run rustfmt after this, use `cargo run --bin rustfmt -- filename`. See the
 notes above on running rustfmt.
 
+All command should be run in the project's root directory.
+
```

Comment:
I don't think this is correct. You should be able to install the nightly toolchain and components from anywhere.

```suggestion
```

Reply:
I have completly re-written the section.